### PR TITLE
feature/employee-can-see-rewards

### PR DESCRIPTION
**Pull request by rniedziolka**

### CODE REVIEW

File: app/controllers/rewards_controller.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    @rewards = Reward.all
+  end
+
+  def show
+    @reward = Reward.find(params[:id])
+  end
+
+  private
+
+  def reward_params
```
Comment (ciejj) [inline app/controllers/rewards_controller.rb]: do we need this in current implementation?

**Review by ciejj — APPROVED** [review]
Looks good! Good Job

File: app/controllers/rewards_controller.rb (hunk 1 above)
Comment (rniedziolka) [reply app/controllers/rewards_controller.rb]: You're right, I copied it from the admin controller, no sense, corrected.

File: spec/support/devise.rb [hunk 2]
```diff
@@ -2,4 +2,5 @@
 
 RSpec.configure do |config|
   config.include Warden::Test::Helpers
```
Comment (oskarlakner) [inline spec/support/devise.rb:4]: you probably don't need both :). Read please docs and search which one is truly needed

**Review by oskarlakner — APPROVED** [review]
🙌

File: spec/support/devise.rb (hunk 2 above)
Comment (rniedziolka) [reply spec/support/devise.rb:4]: Hey, how about that?

https://gist.github.com/boddhisattva/f14d2e860dcd2d45f805eb698b99605f

Specify the below helpers in the RSpec.configure do |config| block of your rails_helper.rb to make sure that the sign_in method is used to login the user into your app before testing the specific feature of your app which requires a signed in user

```
config.include Devise::Test::IntegrationHelpers, type: :system
config.include Warden::Test::Helpers
```

**Pull request merged**